src/android/InAppBrowser.java

```diff
@@ -1184,7 +1190,9 @@ public boolean shouldOverrideUrlLoading(String url, String method) {
             boolean useBeforeload = false;
             String errorMessage = null;
 
-            if(beforeload.equals("yes")
+            if (beforeload.equals("yes") && method == null) {
+                useBeforeload = true;
+            }else if(beforeload.equals("yes")
```

Comment:
Does this mean `method` can be `null`?

Reply:
Yes, For Android 6 and below `shouldOverrideUrlLoading(WebView webView, String url)` is called and it calls this method with `method = null`.